### DISCUSSION TITLE
How can I fill all drives except one?

### DISCUSSION THREAD

**Discussion by 8ullyMaguire** [trapexit/mergerfs#1141] [Ideas]
I've added a new drive before the rest where complete and now it's writing in the new drive but I want to completely use the rest first. Should I use another tool to distribute the contents or just change the setting from the default `category.create=epmfs` to `category.create=eplfs` so that new files actually finish filling the drives? Are there any other settings that could be useful in this case?

**Comment by trapexit** [comment]
1) I need to understand your high level goals to make a proper suggestion. Why do you want to create on old branches only?
2) If you are already using epmfs it wouldn't be writing to the empty path unless you are creating paths manually or started creating paths at the relative root path.
3) Yes, if you want to change how the branches are selected then you need to change the create policy. prioritizing least free space vs most free space will reverse the priority based on free space. If that's what you want then, yes, use that.
